must-gather/gather_addon_resources

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -x
+
+BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
+mkdir -p $BASE_COLLECTION_PATH
+
+DEFAULT_NS=redhat-nvidia-gpu-addon
+ADDON_NAMESPACE=${ADDON_NAMESPACE:-$(oc get subscriptions.operators.coreos.com --field-selector metadata.name=nvidia-gpu-addon-operator -A -ojsonpath={.items[].metadata.namespace} --ignore-not-found)}
+if [[ -z "$ADDON_NAMESPACE" ]]; then
+    echo "Warning: Addon namespace cannot be determined as the subscription was not found, will collect from the default namespace '$DEFAULT_NS'" >&2
+    ADDON_NAMESPACE=$DEFAULT_NS
+fi
+
+oc get ns $ADDON_NAMESPACE > /dev/null
```

Comment:
Minor comment, just adding it here in case we were interested to enforce a bash styleguide (of any kind :) ). 

I have found useful over the years [this](https://github.com/koalaman/shellcheck) tool (both in CIs and while developing locally). In this case, it [suggests](https://github.com/koalaman/shellcheck/wiki/SC2181) something like the following:

```sh
if ! oc get ns "${ADDON_NAMESPACE}" > /dev/null; then
    exit 1
fi
```

I'm not sure which version of these lines I find more readable though, So I leave it up to you :)